forest/geo.py

```diff
@@ -52,6 +54,8 @@ def stretch_image(lons, lats, values, plot_height=None, plot_width=None):
     else:
         raise Exception("Either 1D or 2D lons/lats")
 
+    extend_180(gx, lons)
```

Comment:
In-place call with a function that returns a modified copy. I'm happiest with the modified copy style so this should change to be `gx = extend_180(gx, lons)`. That said, I've not tested the code so it may behave differently to my reading of it.